AdobeStockAsset/Test/_files/objects/asset.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
I don't think we need this fixture as there is more advanced `AdobeStockAsset/Test/_files/asset.php` introduced